app/main.py

```diff
@@ -1,6 +1,35 @@
+import os
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    if api_key := os.getenv("API_KEY"):
+        print("Performing request to Weather API for city Paris ...")
+
+        url = "http://api.weatherapi.com/v1/"
+        api_method = "current.json"
+        city = "Paris"
+
+        response = requests.get(f"{url}{api_method}?key={api_key}&q={city}")
+
+        response_location = response.json()["location"]
```

Comment:
Instead of calling `response.json()` multiple times, store the result in a variable to avoid redundant parsing of the JSON response. This will make the code more efficient.